manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
+	gnome-menus \
+	gnome-system-monitor \
 	nautilus \
+	sushi \
+	eog \
+	evince \
```

Comment:
I don't think we need this.

Reply:
The primary function of the desktop would be doing some troubleshooting and configuration tasks the user can't do with the BPM or GamepadUI. Those mostly involve editing files, looking some documents and maybe images. The thumbnailer is just so it doesn't look like Windows 95.

Reply:
I don't think anyone will need a PDF viewer, which is what evince is.

Reply:
> > I think this background looks better:
> > ![chimeraos_background2](https://user-images.githubusercontent.com/1140312/184436019-70b0a5a1-097f-4231-a486-c7f3d7e1cdb1.png)
> 
> I do too. But since we are mostly be used on OLED or IPS displays I would like to minimize power consumption.

I don't think that is a huge issue. What I don't like about the black background is it is a usability issue because I cannot see the top menu bar.

Reply:
But is great to not see the top menu bar... I like how it blends. Let's do this: add this one too and set as default and have the full black one as an option to the user